base/applications/regedit/find.c

```diff
@@ -195,6 +195,9 @@ BOOL RegFindRecurse(
 
     qsort(ppszNames, c, sizeof(LPWSTR), compare);
 
+    if (pszValueName == s_empty)
```

Comment:
It looks to me that you could even delete the previous `if (pszValueName == NULL) ...` case (that makes the pointer point to `s_empty`, and directly change here your new 'if'-case by `if (pszValueName == NULL)  pszValueName = ppszNames[0];`.